lib/set-delegates.js

```diff
@@ -206,6 +227,24 @@ module.exports = function(browserWindow, panel, webview, options) {
     })
   }
 
+  var themeObserver = ThemeObserverClass.new({
+    utils: {
+      executeJavaScript: browserWindow.webContents.executeJavaScript.bind(
+        browserWindow.webContents
```

Comment:
is it possible to simply pass the `browserWindow` and use `call`? I don't remember exactly how `ObjCClass` handles construction / closures. I do remember this `utils` paradigm from other places in the code-base though, so I'm probably just wishful thinking :)

Reply:
so `browserWindow` is a javascript object while `utils` will be an NSDictionary. I remember some cases where passing a javascript object directly broke Sketch but I can't remember the details now.
I just reuse the same pattern everywhere now hahaha

Reply:
ah yeah - makes sense :)